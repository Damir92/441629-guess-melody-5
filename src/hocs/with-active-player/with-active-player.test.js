import React, {Fragment} from 'react';
import renderer from 'react-test-renderer';
import PropTypes from 'prop-types';

import withActivePlayer from './with-active-player';

const MockComponent = (props) => {
  const {children} = props;

  return (
    <Fragment>
      {children}
    </Fragment>
  );
};

MockComponent.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired,
};

const MockComponentWrapped = withActivePlayer(MockComponent);

it(`withActivePlayer rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped>
      <Fragment />
    </MockComponentWrapped>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
