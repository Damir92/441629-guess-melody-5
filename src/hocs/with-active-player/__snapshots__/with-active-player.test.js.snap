// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActivePlayer rendered correctly 1`] = `null`;
